pkg/vm/evm_prompts.go

```diff
@@ -877,51 +877,41 @@ func promptPermissioning(
 	return params, nil
 }
 
-func PromptVMVersion(
+func PromptSubnetEVMVersion(
```

Comment:
not sure if this is a prompt function. the caller should use a separate function to get the version, and if not obtained, call a prompt function